packages/gatsby/src/redux/__tests__/__snapshots__/pages.js.snap

```diff
@@ -207,7 +207,7 @@ Map {
     "componentChunkName": "component---whatever-2-index-js",
     "context": Object {},
     "internalComponentName": "ComponentHi",
-    "isCreatedByStatefulCreatePages": undefined,
+    "isCreatedByStatefulCreatePages": false,
```

Comment:
this feels like it's likely incorrect. Adding types does not seem like it should change tests

Reply:
This is the result of this diff:
```diff
 isCreatedByStatefulCreatePages:
-     actionOptions &&
+     !!actionOptions &&
      actionOptions.traceId === `initial-createPagesStatefully`
```

Note that changing it to

```
actionOptions?.traceId === "initial-createPagesStatefully"
```

as you suggest below would also lead to changed snapshot. This is because `actionOptions` is either `undefined` or not.